test/tests/concrete_moisture_heat_transfer/tests

```diff
@@ -9,47 +9,596 @@
     rel_err = 1e-6
     abs_zero = 1e-10
   []
-  [thermal_properties]
-    issues = '#9'
-    requirement = 'Blackbear shall compute material properties for thermal transport in concrete '
-    [kodur_2004]
-      detail = 'using the Kodur 2004 model, '
+  issues = '#245'
+  design = 'ConcreteThermalMoisture.md'
+  [thermal_model]
+    requirement = 'Blackbear shall provide the moisture capacity and moisture diffusivity in domain as demonstrated by'
```

Comment:
Blackbear shall compute the heat capacity and thermal conductivity for concrete for the following models and options: